src/gui/gui.config

```diff
@@ -120,6 +120,22 @@
   </ignition-gui>
 </plugin>
 
+<!-- Insert lights -->
+<plugin filename="Lights" name="Lights">
+  <ignition-gui>
+    <anchors target="Shapes">
+      <line own="left" target="right"/>
+      <line own="top" target="top"/>
+    </anchors>
+    <property key="resizable" type="bool">false</property>
+    <property key="width" type="double">250</property>
+    <property key="height" type="double">50</property>
+    <property key="state" type="string">floating</property>
+    <property key="showTitleBar" type="bool">false</property>
+    <property key="cardBackground" type="string">#666666</property>
+  </ignition-gui>
+</plugin>
```

Comment:
This currently looks like this:

![image](https://user-images.githubusercontent.com/5751272/108266548-17abec00-711f-11eb-85e3-2aba10a1b952.png)

How about not using the anchors (because of https://github.com/ignitionrobotics/ign-gui/issues/129), like this:

<details>
  <summary>diff</summary>
  
```.diff
diff --git a/src/gui/gui.config b/src/gui/gui.config
index fe87f0d5..8f1b33a4 100644
--- a/src/gui/gui.config
+++ b/src/gui/gui.config
@@ -91,12 +91,10 @@
 <plugin filename="TransformControl" name="Transform control">
   <ignition-gui>
     <title>Transform control</title>
-    <anchors target="3D View">
-      <line own="left" target="left"/>
-      <line own="top" target="top"/>
-    </anchors>
     <property key="resizable" type="bool">false</property>
-    <property key="width" type="double">230</property>
+    <property key="x" type="double">0</property>
+    <property key="y" type="double">0</property>
+    <property key="width" type="double">250</property>
     <property key="height" type="double">50</property>
     <property key="state" type="string">floating</property>
     <property key="showTitleBar" type="bool">false</property>
@@ -107,12 +105,10 @@
 <!-- Insert simple shapes -->
 <plugin filename="Shapes" name="Shapes">
   <ignition-gui>
-    <anchors target="Transform control">
-      <line own="left" target="right"/>
-      <line own="top" target="top"/>
-    </anchors>
     <property key="resizable" type="bool">false</property>
-    <property key="width" type="double">250</property>
+    <property key="x" type="double">250</property>
+    <property key="y" type="double">0</property>
+    <property key="width" type="double">150</property>
     <property key="height" type="double">50</property>
     <property key="state" type="string">floating</property>
     <property key="showTitleBar" type="bool">false</property>
@@ -123,12 +119,10 @@
 <!-- Insert lights -->
 <plugin filename="Lights" name="Lights">
   <ignition-gui>
-    <anchors target="Shapes">
-      <line own="left" target="right"/>
-      <line own="top" target="top"/>
-    </anchors>
     <property key="resizable" type="bool">false</property>
-    <property key="width" type="double">250</property>
+    <property key="x" type="double">400</property>
+    <property key="y" type="double">0</property>
+    <property key="width" type="double">150</property>
     <property key="height" type="double">50</property>
     <property key="state" type="string">floating</property>
     <property key="showTitleBar" type="bool">false</property>
```

</details>

It looks like this:

![image](https://user-images.githubusercontent.com/5751272/108267135-d1a35800-711f-11eb-8633-151a91dfe947.png)